src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+
+    margin: 0;
+    margin-left: 4px;
+
+    background-image: url(/images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:first-child {
+      margin-left: 0;
+    }
+
+    &:nth-child(n+5) {
+      background-image: url(/images/star.svg);
+    }
+  }
+}
```

Comment:
You need to copy stars from our previous H/W "stars'',  if somebody will change class "stars stars--4" to "stars stars--2" for example, it must change active stars count in every card from 4 to 2.

Reply:
Fixed!